days/src/day10/Day10.kt

```diff
@@ -19,9 +19,7 @@ class Day10(values: List<Int>) : Day {
             }
         }.last()
     }
-    override fun puzzle2(): Int? = jolts.map { it.toLong() }.joltArrangements().also {
-        println("Solution: $it")
```

Comment:
Don't remove this. Since it's a long the answer that gets printed is - so I print the solution before it returns

..I know, hacky